eng/packages/http-client-csharp/generator/TestProjects/Local/Mgmt-TypeSpec/src/MgmtTypeSpec.csproj

```diff
@@ -10,14 +10,33 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.ClientModel" Version="1.1.0-beta.4" />
-    <PackageReference Include="System.Text.Json" Version="8.0.5" />
-		<PackageReference Include="Azure.Core" Version="1.43.0" />
+    <PackageReference Include="System.ClientModel" />
+    <PackageReference Include="System.Text.Json" />
+	  <PackageReference Include="Azure.Core" />
+	  <PackageReference Include="Azure.ResourceManager" />
```

Comment:
Should be part of NewProjectScaffolding for Arm libraries.

Reply:
updated